pkg/gui/controllers/helpers/refs_helper.go

```diff
@@ -271,6 +272,57 @@ func (self *RefsHelper) CreateGitResetMenu(ref string) error {
 	})
 }
 
+func (self *RefsHelper) CreateCheckoutMenu(commit *models.Commit) error {
+	branches := lo.Filter(self.c.Model().Branches, func(branch *models.Branch, _ int) bool {
+		return commit.Hash == branch.CommitHash && branch.Name != self.c.Model().CheckedOutBranch
+	})
+
+	ref := commit.Hash
+	var menuItems []*types.MenuItem
+
+	menuItems = append(menuItems, &types.MenuItem{
+		LabelColumns: []string{fmt.Sprintf("%s %s", self.c.Tr.CheckoutCommit, self.c.Tr.AsDetachedHead), style.FgRed.Sprint(ref[:8])},
+		OnPress: func() error {
+			self.c.Confirm(types.ConfirmOpts{
+				Title:  self.c.Tr.CheckoutCommit,
+				Prompt: self.c.Tr.SureCheckoutThisCommit,
+				HandleConfirm: func() error {
+					self.c.LogAction(self.c.Tr.Actions.CheckoutCommit)
+					return self.CheckoutRef(ref, types.CheckoutRefOptions{})
+				},
+			})
+			return nil
+		},
+		Tooltip: self.c.Tr.CheckoutCommitTooltip,
```

Comment:
I don't feel this tooltip is necessary; it adds almost no extra information beyond what the title of the menu entry already says.

Reply:
Should I just remove it or would you recommend a better description?

Reply:
I'd just remove them. (I see that you did already.)